packages/access-om3-virtual/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# Copyright 2024 ACCESS-NRI
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import BundlePackage, version, depends_on
+
+
+class AccessOm3Virtual(BundlePackage):
+    """
+    ACCESS-OM3 bundle containing the ACCESS-OM3 Package.
+
+    This is used to version the entirety of a released deployment, including
+    the package, it's dependencies, and the version of
+    spack-packages/spack-config that it is bundled with
+    """
+
+    homepage = "https://www.access-nri.org.au"
+
+    git = "https://github.com/ACCESS-NRI/ACCESS-OM3.git"
+
+    maintainers = ["harshula"]
+
+    version("latest")
+
+    # FIXME: How do we inject variants into the package from the SBD?
```

Comment:
Example from `access-om2`:
```
    variant("deterministic", default=False, description="Deterministic build.")

    depends_on("libaccessom2+deterministic", when="+deterministic")
    depends_on("libaccessom2~deterministic", when="~deterministic")
    depends_on("cice5+deterministic", when="+deterministic")
    depends_on("cice5~deterministic", when="~deterministic")
    depends_on("mom5+deterministic", when="+deterministic")
    depends_on("mom5~deterministic", when="~deterministic")
```

Reply:
Although I'm wondering if we need to have those variants in there, since they can be specified in the `spack.yaml` like so:
```yaml
spack:
  specs:
    - access-om3-virtual@git.2024.04.0
  packages:
    access-om3:
      require:
        - '@git.v0.2.0'
        - '<variants here>
    # more packages...
```

Reply:
For non-environment installs the SBD would need the variants that you insert in the spack.yaml. It's best to keep them synchronised until we have a better option.